docs/getting-started-guides/docker-multinode/testing.md

```diff
@@ -63,14 +63,13 @@ now run `docker ps` you should see nginx running.  You may need to wait a few mi
 kubectl expose rc nginx --port=80
 ```
 
-This should print:
+Run the following command to obtain the IP of this service we just created. There are two IPs, the first one is internal, and the second one is the external load-balanced IP.
 
-```console
-NAME         CLUSTER_IP       EXTERNAL_IP       PORT(S)                SELECTOR     AGE
-nginx        10.179.240.1     <none>            80/TCP                 run=nginx    8d
+```sh
+kubectl get svc nginx
```

Comment:
It might be a bit advanced as a getting started guide. I'll include this as an alternative.